components/Home/HomeTabs.js

```diff
@@ -78,7 +78,7 @@ export const HomeTabs = ({ person, onChange, defaultTab }) =>
     </TabPane>
     <TabPane tab={homeProfileTab} key='profile'>
       <ProfilePanel>
-        <EditablePersonPanel person={person} />
+        <EditablePersonPanel person={person} me={me} />
```

Comment:
it is not necessary to pass 'me' down through multiple levels of properties.  me is in the redux session so it is available everywhere. 
use the hook.
const me = useSelector(state => state.session.me)

Reply:
Sounds good I'll change this

Reply:
Looks like useSelector is only for NFC, PersonDetailForm is a class so it doesn't like being there
Anyway to access the redux store directly?

Reply:
export default connect(store => ({  me: store.session.me } )(PersonDetailForm)
should do it.

Reply:
Make the existing default export into a named const and wrap that.

Reply:
Thanks :) That's worked nicely :) I've only used redux a little bit, it solves this case of piping props down and down nicely :)